controller/resource.go

```diff
@@ -42,11 +44,16 @@ func (c *ResourceController) Read(ctx *app.ReadResourceContext) error {
 // Register runs the register action.
 func (c *ResourceController) Register(ctx *app.RegisterResourceContext) error {
 
-	// TODO validate the PAT here
+	// Validate the PAT
+	_, err := c.TokenManager.Locate(ctx)
+	if err != nil {
+		jerrors, _ := jsonapi.ErrorToJSONAPIErrors(ctx, goa.ErrBadRequest(err.Error()))
+		return ctx.BadRequest(jerrors)
```

Comment:
We have started using the following format everywhere
`jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError(...pass the err object))`